### PR TITLE
 Azs.Fabric.Admin module generation, metadata changes, customization and test

### PR DESCRIPTION
**Pull request by LingyunSu**
## Description

Azs.Fabric.Admin module with Az
1. Directives and customization
2. Docs and examples

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by LingyunSu** [comment]
This is phase 1 update, since too many files, this phase only including the Directives and customization + Docs and examples. Tests will be done in coming update.

**Comment by bganapa** [comment]
@LingyunSu Can you please remove the examples and docs from this PR. They are simply empty or place holders anyway. You can make a separate PR for docs and examples when ready

File: src/StackAdmin/Azs.Fabric.Admin/readme.md [hunk 1]
```diff
@@ -173,6 +230,34 @@ directive:
...
+  - where:
+      subject: StorageOperationResult
+    hide: true
+
+  # Hide the auto-generated New-AzsScaleUnitFromJson and expose it through customized one
+  - where:
+      subject: ScaleUnitFromJson
+    hide: true
+
 subject-prefix: ''
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/readme.md]: The changes in the readme.md looks good to me

File: src/StackAdmin/Azs.Fabric.Admin/readme.md [hunk 2]
```diff
@@ -76,6 +91,48 @@ directive:
       default:
         script: -join("System.",(Get-AzLocation)[0].Name)
```
Comment (rakku-ms) [inline src/StackAdmin/Azs.Fabric.Admin/readme.md]: use (Get-AzLocation)[0].Location instead of (Get-AzLocation)[0].Name

**Comment by markcowl** [comment]
@LingyunSu  Please respond to @bganapa review comments

File: src/StackAdmin/Azs.Fabric.Admin/docs/Get-AzsStoragePool.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
+---
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/docs/Get-AzsStoragePool.md]: Need to add content in the help files

File: src/StackAdmin/Azs.Fabric.Admin/readme.md [hunk 4]
```diff
@@ -74,14 +89,161 @@ directive:
...
-        script: -join("System.",(Get-AzLocation)[0].Name)
+        script: -join("System.",(Get-AzLocation)[0].Location)
+
+  # [EdgeGateway]: Folloing changes are for EdgeGateway
+  # [EdgeGateway] Propertity Rename: change NumberOfConnection to NumberOfConnections
+  - where:
+      model-name: EdgeGateway
+      property-name: NumberOfConnection
+    set:
+      property-name: NumberOfConnections
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/readme.md]: should this change be done in the swagger ?

**Pull request merged**